### PR TITLE
Dynamic data totals for dashboard

### PR DESCRIPTION
**Pull request by evan-toyberg**
Ready for review

### CODE REVIEW

**Comment by evan-toyberg** [comment]
This is now ready for review

File: mirrulations-dashboard/src/mirrdash/static/index.js [hunk 1]
```diff
@@ -97,21 +94,26 @@ const updateClientDashboardData = () => {
...
+            regulations_total_dockets,
+            regulations_total_documents,
+            regulations_total_comments
         } = jobInformation;
+
+        let regulations_totals = regulations_total_dockets + regulations_total_documents + regulations_total_comments
+
         updateHtmlValues(num_jobs_waiting, num_jobs_done);
-        updateCorpusProgressHtml([num_dockets_done, num_documents_done, num_comments_done, num_attachments_done], [232255, 1718669, 18072106, 15000000]); //TO DO: change hard coded numbers
+        updateCorpusProgressHtml([num_dockets_done, num_documents_done, num_comments_done, num_attachments_done], regulations_totals)
```
Comment (jack11wagner) [inline mirrulations-dashboard/src/mirrdash/static/index.js:105]: Looks good

File: mirrulations-dashboard/src/mirrdash/static/index.js [hunk 2]
```diff
@@ -97,21 +94,26 @@ const updateClientDashboardData = () => {
...
         // Counts for percents
-        updateJobTypeProgress("dockets-done",num_dockets_done, 232255);
+        updateJobTypeProgress("dockets-done", num_dockets_done, regulations_total_dockets);
+        updateJobTypeProgress("documents-done",num_documents_done, regulations_total_documents);
+        updateJobTypeProgress("comments-done",num_comments_done, regulations_total_comments);
         // Current estimate of number of attachments (from comments)
         updateJobTypeProgress("attachments-done",num_attachments_done, 15000000);
```
Comment (jack11wagner) [inline mirrulations-dashboard/src/mirrdash/static/index.js:111]: I had an idea for this 
if we did
`(num_attachments_done / num_comments_done)* regulations_total_comments`
for `total_attachments`

Comment (evan-toyberg) [reply mirrulations-dashboard/src/mirrdash/static/index.js]: Yeah, good idea, that's added in. It displays NaN in a local dev environment (divide by 0) but I figured that's a better default value than 0 or an arbitrary number.

Comment (AlexF301) [reply mirrulations-dashboard/src/mirrdash/static/index.js]: Works Properly when not dividing by 0! Good call guys.

**Comment by jack11wagner** [comment]
Excellent, Merging now.

**Pull request merged**